conftest.py

```diff
@@ -42,11 +42,10 @@ def setup_session_scope(
 def setup_function_scope(
         caplog,
         generate_test_data,
-        check_result,
-        application_logs
+        check_result
 ):
     # FIXME: broken due to KeyError: <_pytest.stash.StashKey object at 0x7fd1ba6d78c0>
-    #caplog.set_level(configs.LOG_LEVEL)
+    # caplog.set_level(configs.LOG_LEVEL)
```

Comment:
Is it supposed to be commented?

Reply:
i don't yet know why we need that at all 😂 It was like that, i did not change it